projects/github.com/darthsim/overmind/package.yml

```diff
@@ -0,0 +1,37 @@
+distributable:
+  url: https://github.com/DarthSim/overmind/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: overmind
+
+versions:
+  github: darthsim/overmind
+
+platforms:
```

Comment:
if it doesn't work on `linux/aarch64`, please put in a comment as to why. usually `go` bin are pretty easy to build.

Reply:
oops, not intentional. had just copied from another formula